app/library/postgres.py

```diff
@@ -209,6 +209,16 @@ async def delete_scene_from_postgres(scene_id: str, session):
     if not delete_scene(scene_id, session):
         raise ValueError("Scene ID not valid")
 
+    # Remove the scene from scenario's scene_ids array
+    scenarios = get_scenarios(session)
+    for scenario in scenarios:
+        if int(scene_id) in scenario.scene_ids:
+            scenario_obj: Scenario = await get_scenario_from_postgres(
+                scenario.id, session
+            )
+            scenario_obj["scene_ids"].remove(int(scene_id))
```

Comment:
`scenario_obj` is not a dict, on line `scenario_obj: Scenario `. So you should use `scenario_obj.scene_ids` if it is a scenario, OR change the above line to  `scenario_dict: dict`. But like the previous comment mentions, scenario_obj is redundant.  
![image](https://user-images.githubusercontent.com/46459060/118756095-69036e80-b827-11eb-964a-cf0b1185c17f.png)